AppController/djinn.rb

```diff
@@ -6324,15 +6184,23 @@ def get_all_stats(secret)
       }
     end
 
-    all_stats["public_ip"] = my_node.public_ip
-    all_stats["private_ip"] = my_node.private_ip
-    all_stats["roles"] = my_node.jobs or ["none"]
-    Djinn.log_debug("All stats: #{all_stats}")
+    node_stats["cloud"] = my_node.cloud
```

Comment:
@cdonati , yes, I guess - I saw `cloud1` as its value. I checked usages of this and found that such refactoring may affect number of tests and probably some functionality. So it may require a couple of extra builds which would find all hidden dependencies. So I wouldn't remove `cloud` in this branch.